tests/application-operator-tests/test/testkit/config.go

```diff
@@ -19,7 +19,7 @@ const (
 type TestConfig struct {
 	Namespace           string
 	TillerHost          string
-	ProvisioningTimeout int
+	InstallationTimeout int
```

Comment:
This field is always representing the time in seconds, changing it to `InstallationTimeoutSeconds` would improve readability. What do you think?

Reply:
Same goes for TestSuite structs in other files, it would be best to change `installationTimeoutEnvName` value to something like `INSTALLATION_TIMEOUT_SECONDS` but if that is problematic (due to chart changes and upgradeability) let's just leave those like they are now.